explorer/src/__tests__/realtime.test.ts

```diff
@@ -195,26 +195,32 @@ describe('realtime', () => {
   })
 
   it('rejects multiple connections from single node', async done => {
-    expect.assertions(4)
+    expect.assertions(8)
 
-    const ws1: WebSocket = await newChainlinkNode(
-      ENDPOINT,
-      chainlinkNode.accessKey,
-      secret,
-    )
-    const ws2: WebSocket = await newChainlinkNode(
-      ENDPOINT,
-      chainlinkNode.accessKey,
-      secret,
-    )
+    // eslint-disable-next-line prefer-const
+    let ws1: WebSocket, ws2: WebSocket, ws3: WebSocket
+
+    // eslint-disable-next-line prefer-const
```

Comment:
Is this required?

Reply:
Both comments are necessary to prevent the linter from fussing with the code as is.

The comment on line 200 is definitely necessary because the variables `ws2` and `ws3` have to be initialized before they are defined so that they can be closed over in the `close` callbacks. The comment on 203 I *think* could be removed if we changed the code to loo like this...

```ts
    // eslint-disable-next-line prefer-const
    let ws2: WebSocket, ws3: WebSocket

    const ws1 = await newChainlinkNode(ENDPOINT, chainlinkNode.accessKey, secret)

    ws1.addEventListener('close', (event: WebSocket.CloseEvent) => {
      // ...
    }

    ws2 = await newChainlinkNode(ENDPOINT, chainlinkNode.accessKey, secret)
    //...
```

But I thought the test would be more readable if all the sockets were defined in the same way. What do you think? I'm okay going either route.. just wanted to make the job other developers as easy as possible :smile:.

Reply:
👍 I had just assumed it would only catch the `let` definition. But interesting to know it's smart enough to pick up the assignment also.